packages/synapse-react-client/src/components/TermsAndConditions/TermsAndConditions.tsx

```diff
@@ -1,116 +1,74 @@
-    label: 'I will support open access best practices for public facing data.',
-    description: `The Synapse open access platform promotes data accessibility and collaboration. We encourage you to 
-    contribute your research findings to open access journals when applicable.<br><br>
-    You agree to support open access best practices when possible (e.g., sharing code, metadata, annotating files for discovery).`,
-  },
-  {
-    icon: <PenIcon />,
-    label: 'I will credit research participants and all data sources.',
-    description: `You agree to acknowledge data participants and to cite contributors and data sources using the language provided with the dataset.<br><br>
-    It is our policy to terminate the access privileges of those who infringe the intellectual property rights of others. For more information, please refer to our full 
-    <a target="_blank" href=${URL_TERMS_CONDITIONS_AGREEMENT}>Terms and Conditions of Use</a>.`,
-  },
-  {
-    icon: <SpeakerIcon />,
-    label: 'I will not use data for marketing and/or advertising.',
-    description: `Data may not be used for marketing or advertising purposes. You agree not to upload or otherwise transmit 
-    any unsolicited or unauthorized advertising, promotional materials, junk mail, spam, or any other form of solicitation (commercial or otherwise). <br><br>
-    Please review the <a target="_blank" href="https://s3.amazonaws.com/static.synapse.org/governance/SynapsePrivacyPolicy.pdf?v=5">Synapse Privacy Policy</a> for Sage's commitment to privacy protections.`,
-  },
-  {
-    icon: <FlagIcon />,
-    label: `I will report any suspected data breach or misuse to privacyofficer@sagebase.org within 2 business days of initial incident knowledge.`,
-    description: `Data misuse includes violating dataset access requirements, unauthorized use or 
-    redistribution, lack of data accreditation, and unethical data handling.<br><br>
-    For more information about what classifies as a violation of our data governance policies, please refer to our 
-    <a target="_blank" href=${dataUseLink}>Data Use Procedure</a> and <a target="_blank" href=${URL_TERMS_CONDITIONS_AGREEMENT}>Terms and Conditions of Use</a>.`,
-  },
-]
-
 const TermsAndConditions: React.FunctionComponent<TermsAndConditionsProps> = ({
   onFormChange,
   hideLinkToFullTC = false,
 }) => {
-  const checkboxCount = tcList.length
+  const [tcList, setTcList] = useState<tcItem[]>([])
+  // Fetch the table data
+  const tableQuery = useGetFullTableQueryResults({
+    entityId: termsAndConditionsTableID,
+    query: {
+      sql: `SELECT * FROM ${termsAndConditionsTableID} ORDER BY order asc`,
+    },
+    partMask: BUNDLE_MASK_QUERY_RESULTS,
+    concreteType: 'org.sagebionetworks.repo.model.table.QueryBundleRequest',
+  })
+  const { data, isLoading } = tableQuery
+
+  // update tcList when data changes (transform)
+  useEffect(() => {
+    if (data && data.queryResult && data.queryResult.queryResults) {
```

Comment:
Do the transform into tcItem list, and update the list.

Reply:
It looks like `select` expects a function that returns the same type, in this case
`(data: QueryResultBundle) => QueryResultBundle`
I would need to do some hacky things to make this a QueryResultBundle, so keeping the tcList for now